edx_credentials_themes/__init__.py

```diff
@@ -1 +1,2 @@
-__version__ = '0.3.3'
+__version__ = '1.2.0'
```

Comment:
maybe this should be `0.4.0` and not `1.x.x`?

Reply:
Updated to match the suggested `0.4.0`.